test/routes/form.test.js

```diff
@@ -649,8 +649,9 @@ describe('GET /ratings/violence', () => {
       .expect('Content-Type', /html/)
       .expect(res => {
         expect(res.text).toMatch(/Home.+Categorisation home.+Safety and good order/s)
-        expect(res.text).toContain(`This person has been reported as the perpetrator in 5 assaults in custody before,
-      including 2 serious assaults and 3 non-serious assaults in the past 12 months.`)
+        expect(res.text).toMatch(
+          /This person has been reported as the perpetrator in 5 assaults in custody before,\s+including 2 serious assaults and 3 non-serious assaults in the past 12 months./
+        )
```

Comment:
these changes were needed to get tests working on windows (where there is a CR in the string)